packages/web3torrent/src/components/torrent-info/download-info/DownloadInfo.tsx

```diff
@@ -29,18 +29,26 @@ const DownloadInfo: React.FC<DownloadInfoProps> = ({
   const totalSpent = myPayingChannelIds
     .map(id => channelCache[id].beneficiaryBalance)
     .reduce((a, b) => bigNumberify(a).add(bigNumberify(b)), bigNumberify(0));
+
+  const displayProgress = !(torrent.done || torrent.paused);
```

Comment:
I think we need to consider what the user sees in `.paused`. Right now, it's not clear that they are in that state. It would be nice to let them know that
- They didn't finish the download
- They may have still paid some money out

Although (as we discussed) the low level `pause` method is better than `remove` (when the user clicks [ Cancel Download ] ), from a UI perspective the torrent isn't really paused. Pausing suggests the possibility of restarting, we which aren't currently supporting.

Reply:
Actually, the status of the torrent is shown as `Cancelled`.

The UI looks like below after Cancel Download is pressed:
![image](https://user-images.githubusercontent.com/2715151/77325163-35de5180-6cee-11ea-9391-e953959266bd.png)